app/controllers/blacklisted_repos_controller.rb

```diff
@@ -0,0 +1,15 @@
+class BlacklistedReposController < ApplicationController
+  def create
+    current_user.blacklisted_repos.create(blacklisted_repo_name: params[:blacklisted_repo_name])
+    redirect_to dashboard_path
+  end
+
+  def destroy
+    current_user.blacklisted_repos.find_by(blacklisted_repo_name: params[:blacklisted_repo_name])&.destroy
```

Comment:
to powinien byc find

Reply:
no i znowu muszę się nie zgodzić z Twoją teorią bo tutaj wyszukujemy po repo_name a nie po id co czego przeznaczony jest właśnie find ;)